labs/oas-apigee-mock/README.md

```diff
@@ -0,0 +1,11 @@
+# Apigee OAS Mock Lab
+
```

Comment:
Maybe add a sentence on what this is all about. Happy if you want to just copy the one from the main readme. I think this captures it nicely.